Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -124,6 +127,7 @@ void JitArm64::ClearCache()
   m_handler_to_loc.clear();
 
   blocks.Clear();
+  Common::ScopedJITPageWriteAndNoExecute enable_jit_page_writes;
```

Comment:
Bikeshedding: I know the struct has no data, but would it make sense to make these const anyhow? When I see a non-const variable it makes me wonder if it gets modified somewhere.

Reply:
Done